### ISSUE TITLE
have a tabletop mod for legendary. maybe collaborate?

### ISSUE THREAD

**Issue by Nightbreez** [matdillen/legendary-generator#7]
i was pointed here by a user of my mod suggesting maybe a way to import decks or something. i was wondering if maybe you could share some thoughts on how to go about doing it. feel free to add me on steam, same username, and you can find my mod on the workshop as well.

**Comment by matdillen** [comment]
Hi, I added you on steam. I'm not sure what you mean by import decks. If you want to import setups, then I may be able to help.

I just published the adapted mod I built from maxlaager's mod. The biggest change is that there's an import function now to automatically deal the cards for a specific setup. Setups are imported as structured text from clipboard.

It's friends only for the moment, as I still need some actual testing by others and in multiplayer to see if there's still big issues. Hope to do that later this week, but feel free to take a look as well.
